ui/src/pages/static-instance.js

```diff
@@ -142,10 +143,16 @@ export default class StaticInstancePage extends React.Component {
                                         <ServiceNameHeader
                                             domain={domain}
                                             service={service}
+                                            serviceHeaderDetails={
+                                                this.props.serviceHeaderDetails
+                                            }
                                         />
                                         <ServiceInstanceDetails
-                                            details={
-                                                this.props.serviceHeaderDetails
+                                            instanceDetailsMeta={
+                                                this.props.instanceDetails
+                                            }
+                                            categoryType={
+                                                SERVICE_TYPE_STATIC_LABEL
```

Comment:
we should use `SERVICE_TYPE_STATIC` instead of `SERVICE_TYPE_STATIC_LABEL`

Reply:
done